include/mbgl/map/map.hpp

```diff
@@ -149,6 +149,7 @@ class Map : private util::noncopyable {
     void removeAnnotation(AnnotationID);
 
     AnnotationIDs getPointAnnotationsInBounds(const LatLngBounds&);
+    AnnotationIDs getShapeAnnotationsInBounds(const LatLngBounds&);
```

Comment:
#5165 landed, so now we can implement a `queryShapeAnnotations()` that calls `queryRenderedFeatures()` – no custom querying logic is needed anymore. Unlike in `queryPointAnnotations()`, the call to `queryRenderedFeatures()` will need to pass in the layer IDs of all the shape annotation layers.